db/generateSeedData.js

```diff
@@ -1,57 +1,56 @@
 const faker = require('faker');
 
-// Generate Item Reviews
-var generateItems = (num) => {
+// Generate Item (num is total number of items), number of reviews is seeded as random for now
+let generateItems = (num) => {
     var items = [];
     for (var i = 0; i < num; i++) {
         var itemObject = {
-            itemId: i,
             itemName: faker.commerce.productName(),
+            numberOfReviews: Math.floor(Math.random() * 10000 + 1),
             price: faker.commerce.price(),
-            averageStars: Math.floor(Math.random() * (5-1)+1),
+            averageStarRating: Math.floor(Math.random() * (5)+1),
             availableOnPrime: (Math.random() < .8),
             image: faker.image.business()
         };
 
         items.push(itemObject);
     }
-
-    console.log(items)
+    // console.log(items);
     return items; 
 };
 
-// Generate Random Reviews
-var generateReviews = (num) => {
+// Generate Random Reviews, creates num reviews over numItems randomly (each review will have a random item number)
+let generateReviews = (num, numItems) => {
     var reviews = [];
     for (var i = 0; i < num; i++) {
         var reviewObject = {
-            text: faker.lorem.sentence(),
+            body: faker.lorem.sentence(),
             headline: faker.lorem.paragraph(),
             photoUrl: faker.image.image(),
-            rating: Math.floor(Math.random() * (5-1)+1),
-            item_id: Math.floor(Math.random() * 99 + 1)
+            rating: Math.floor(Math.random() * (5 - 1) + 1),
+            id_item: Math.floor(Math.random() * (numItems - 1) + 1)
         };
         reviews.push(reviewObject);
     }
-    console.log(reviews);
+    // console.log(reviews);
     return reviews;
 };
 
 // Generate Singles, Pairs or Trios of frequently together items by id 1 - 100
-var generateFrequentlyTogether = (num) => {
+let generateFrequentlyTogether = (num) => {
     var rangeOfItems = [...Array(num + 1).keys()];
     rangeOfItems.shift();
     var returnArr = [];
     while (rangeOfItems.length > 0) {
         if (rangeOfItems.length > 3) {
             var randThroughThree = Math.floor(Math.random() * (3 - 1 ) + 1);
             var tempRandArr = [];
-
             for (var i = 0; i < randThroughThree; i++) {
                 var randomItemIdIndex = Math.floor(Math.random() * (rangeOfItems.length - 1 ) + 1);
-                tempRandArr.push(rangeOfItems[randomItemIdIndex]);
```

Comment:
If rangeOfItems is an array of incrementing integers, wouldn't rangeOfItems[randomItemIdIndex] be an integer like randomItemIdIndex? Why not add randomItemIdIndex itself (or itself + 1)?